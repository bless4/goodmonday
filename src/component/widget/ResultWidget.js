import React from 'react';

export default class ResultWidget extends React.Component {
    render() {
        const { props } = this;
        return(
            <div>
                <p>Result Pair Array : {props.resultArray}</p>
                <p> Array Pair  Multiplication: {props.resultMultiArray}</p>
                <p>Total Sum : {props.resultSum}</p>
            </div>
        )
    }
}