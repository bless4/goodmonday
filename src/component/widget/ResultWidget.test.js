import React from 'react';
import { shallow } from 'enzyme';
import ResultWidget from './ResultWidget';

describe('ResultWidget', () => {
  let component = null;
  
  it('renders correctly', () => {
    component = shallow(<ResultWidget />);
  });

  it('result snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
