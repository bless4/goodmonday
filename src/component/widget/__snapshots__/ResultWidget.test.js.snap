// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultWidget result snapshot 1`] = `ShallowWrapper {}`;
